app/controllers/twilio_webhooks_controller.rb

```diff
@@ -21,6 +21,10 @@ def create_incoming_text_message
       client: client,
     )
     head :ok
+
+  # To notify any users on the client's page that there is new contact info, add this
```

Comment:
This is where we would add code to notify web browsers of a change. If we were worried about load, this could peacefully occur in a job.